src/lib/style/bootstrap4_backports/_overrides.scss

```diff
@@ -76,19 +76,16 @@
 }
 
 // Modals
-.modal-content {
-  padding: 5px 30px;
-  .modal-header {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 10px 0px;
-
-    .modal-title {
-      order: -99;
-    }
-    button.close {
-      order: 99;
+body:not(#body) {
+  // We don't want these styles in Synapse.org, it has its own modal styling
```

Comment:
Should we continue using this pattern going forward?

Reply:
Maybe we should be more explicit.  Perhaps we set the html element in SWC to id="synapse.org"?  Then this would say body:not(#synapse.org)...